frontend/src/components/TabbedResults.less

```diff
@@ -1,36 +1,60 @@
-
 .close {
-  width: auto;
-  margin-left: 1ex;
+    width: auto;
+    margin-left: 1ex;
 }
 
-.tab-content > .tab-pane {
-  margin-top: 2em;
+#tabbed-results {
+    display: flex;
+    flex-direction: column;
+}
+
+.tab-content {
+    flex: auto 1;
+
```

Comment:
I'd group `flex` rules and remove the extra empty line.

Reply:
This is both a flex item (`flex`) and a flex container (`display` & `flex-direction`), I think it makes sense to have them separated.